### PR TITLE
chore(tests): Add PHP 7 to Travis

### PR DESCRIPTION
**Pull request by ewinslow**

### CODE REVIEW

**Comment by ewinslow** [comment]
We will get typed return values eventually!!!

File: composer.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "prefer-stable": true,
     "type": "project",
     "require": {
-        "php": "~5.4",
+        "php": "~5.4 | ~7.0",
```
Comment (Srokap) [inline composer.json]: Why not `>=5.4` ? Is there anything in PHP 6 dev (that was never officially released) that we are afraid of?

Comment (mrclay) [inline composer.json]: Yes, this seems unnecessary. There's no PHP 6.

**Comment by mrclay** [comment]
Anecdote 1: PHPUnit test suites runs in less than half the time on PHP7 Travis build.

**Comment by ewinslow** [comment]
I prefer to explicitly least the major versions we support. Suppose php 8 comes along? Then we have to do something if we're not compliant.

**Comment by ewinslow** [comment]
Feel free to submit a follow up if you feel strongly

**Pull request merged**